library/src/main/java/com/pokegoapi/util/PokeDictionary.java

```diff
@@ -15,27 +15,51 @@
 
 package com.pokegoapi.util;
 
+
 import java.util.Locale;
 import java.util.MissingResourceException;
 import java.util.ResourceBundle;
 
 /**
- * Offers methods to get information about pokémon as seen in the pokédex.
+ * Offers methods to get information about pokemon as seen in the pokedex.
  */
 public class PokeDictionary {
 	private static final String POKE_NAMES_BUNDLE = "pokemon_names";
 	private static final String POKE_DESCRIPTIONS_BUNDLE = "pokemon_descriptions";
 
+	/**
+	 * An array of all supported locales.
+	 */
+	public static final Locale[] supportedLocales = {
+			Locale.GERMAN,
+			Locale.ENGLISH,
+			new Locale("es"),
+			Locale.FRENCH,
+			Locale.ITALIAN,
+			Locale.JAPANESE,
+			Locale.KOREAN,
+			new Locale("ru"),
+			new Locale("zh", "CN"),
+			new Locale("zh", "HK"),
+			new Locale("zh", "TW"),
+	};
+
 	private static ResourceBundle getPokeBundle(String bundleBaseName, Locale locale)
 			throws MissingResourceException {
-		return ResourceBundle.getBundle(bundleBaseName, locale);
+		return ResourceBundle.getBundle(bundleBaseName, locale, new ResourceBundle.Control() {
+			@Override
+			public Locale getFallbackLocale(String baseName, Locale locale) {
+				return Locale.ENGLISH;
```

Comment:
I have changed the Fallback method to return Locale.ENGLISH instead of Locale.getDefault().
That should lessen the exception probability to near zero for Locales that have no pokemon names and/or descriptions.